bapi/resources/bucketlist_items.py

```diff
@@ -0,0 +1,135 @@
+from flask import g
+from flask_restful import Resource, reqparse
+
+from bapi.bucketlist_models import Bucketlist, BucketListItem, DB
+from bapi.verification import token_auth
+
+
+class CreateBucketlistItem(Resource):
+
+    """Creates a new Item in Bucketlist."""
+
+    @token_auth.login_required
+    def post(self, id):
+        try:
+
+            item_query = BucketListItem.query.filter_by(bucketlist_id=id).all()
+            existing_items = [item.name for item in item_query]
```

Comment:
Evaluate this inside the [condition](https://github.com/Bopchy/bucketlist-api/pull/3/files#diff-9d9bcb4dd65e9c485ad14b9621deab6aR26)